src/System.IO.Compression/ref/Compat/project.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
yup, since we already had this namespace in corefx I only created a ref assembly for now to have a full closure of our contracts, but I guess we will eventually have to add a csproj as well.